lib/fathead/mdn_css/output.txt

```diff
@@ -473,6 +698,9 @@ border-bottom-style	A			css properties							<section class="prog__container"><p
 border-bottom-style property	R	border-bottom-style										
 border-bottom-width	A			css properties							<section class="prog__container"><p> </p><p><b>Initial Value: </b><em>medium</em></p><pre><code>/* Any <length> value */\nborder-bottom-width: 10em;\nborder-bottom-width: 3vmax;\nborder-bottom-width: 6px;\n\n/* A keyword */\nborder-bottom-width: thin;\nborder-bottom-width: medium;\nborder-bottom-width: thick;\n\nborder-bottom-width: inherit;\n</code></pre></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/border-bottom-width
 border-bottom-width property	R	border-bottom-width										
+border-box	A										<section class="prog__container"><p>Uses the border box as the reference box.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path#Values
```

Comment:
I think we should ignore the URLs that end in `#Values` this content is out of context and I don't think it will work well...